warmup.krun

```diff
@@ -1,9 +1,12 @@
 import os
+import sys
 from krun.vm_defs import (PythonVMDef, LuaVMDef, JavaVMDef, GraalVMDef,
 		          PHPVMDef, JRubyTruffleVMDef, V8VMDef, NativeCodeVMDef,
 			  find_internal_jvmci_java_bin)
 from krun import EntryPoint
 
+PLATFORM = sys.platform
```

Comment:
This is pointless -- it just obscures what PLATFORM actually is and makes the latter code harder to understand. Let's kill it.